lang/en/package.html

```diff
@@ -17,66 +17,61 @@ <h2 class="details-main--title d-inline-block m-2">…</h2>
             <!-- react-text: 8 -->
             <!-- /react-text -->
           </a><span class="ais-Hit--popular null" title="0 downloads in the last 30 days">0</span>
-          <!-- react-empty: 10 --><span class="ais-Hit--version">0.0.0</span></div>
+          <!-- react-empty: 10 --><span class="ais-Hit--version"></span></div>
```

Comment:
Not really through a script, I just load the view without any package manually and copy the html that's there. 

Do you think it's worth it to write a script?